README.md

```diff
@@ -27,6 +27,7 @@ When using this together with `esp-println` make sure to use the same output kin
 | exception-handler | Include an exception handler, will add `esp-println` as a dependency |
 | print-uart        | Use UART to print messages\*                                         |
 | print-jtag-serial | Use JTAG-Serial to print messages\*                                  |
+| print-defmt       | Use `defmt` logging to print messages\*                              |
```

Comment:
would it make sense to mention it's `defmt-espflash`?

Reply:
We should change a naming for the feature, since we're using `print-*` format for main communication methods here. 
Me and @bjoernQ we decided that `defmt-espflash` sounds reasonable since it's matching `esp_println` feature naming and describes a functionality a bit better.